src/v3/webpack.dev.config.ts

```diff
@@ -107,7 +107,9 @@ const devConfig: Configuration = mergeWithRules({
           loader: 'babel-loader',
           options: {
             plugins: [
-              ...process.env.IE11_COMPAT_MODE === 'true' ? [] : ['@prefresh/babel-plugin'],
+              // Disabled #3 in https://github.com/okta/okta-signin-widget/pull/3356
+              //  because of issue "Maximum call stack size exceeded"
+              // ...process.env.IE11_COMPAT_MODE === 'true' ? [] : ['@prefresh/babel-plugin'],
```

Comment:
Fixes HMR

Reply:
What is the effect here on IE11 testing in playground?

Reply:
Live HMR was not working in IE11 (despite [this note](https://github.com/preactjs/prefresh#usage-in-ie11), I couldn't make it work with IE11 in [this PR](https://github.com/okta/okta-signin-widget/pull/3356))
With `IE11_COMPAT_MODE=true`  plugins `@prefresh/babel-plugin` and `@prefresh/webpack` are disabled (lines 112 , 126).
So this change should have no effect on IE11.

Checked how playground works on IE11 and added a fix: https://github.com/okta/okta-signin-widget/pull/3476/commits/5b74801fab0a044b9afb0bd912170d40f18d48e2#diff-11ee664012ddfbbe9392744c520696a96dd8ebb553f4b64eaaeaf422a6b0ade7R97